README.md

```diff
@@ -1,16 +1,248 @@
-# Title
+# Methods in Ruby
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe routines and procedures for our code using methods
+- Define a method
+- Recognize how to call methods
 
 ## Introduction
 
-## SWBAT 1
+A method in Ruby is a _block_ of code that returns a value. With methods, we can
+organize our code into chunks of reusable functionality that can be easily used
+by other areas of their program.
 
-## SWBAT 2
+Ruby methods are very similar to functions in any other programming language.
+Ruby methods are used to bundle one or more repeatable statements into a single
+unit.
+
+In this lesson we'll introduce methods, distinguish them from data types, and
+cover how to create and execute them in your Ruby program.
+
+## Describe Routines and Procedures for Our Code Using Methods
+
+Methods define a new thing that your program can do. Variables in Ruby store
+data; methods store a new routine or behavior your program can use. Variables
+are like "nouns", a thing, and methods are like "verbs", an action.
+
+For example, imagine needing to say "Hello World!" five times. It could look
+like this:
+
+```ruby
+phrase = "Hello World!"
+puts phrase
+puts phrase
+puts phrase
+puts phrase
+puts phrase
+```
+
+That technically works. You made use of a variable to store the data you wanted
+to output, and then the next five lines output the phrase.
+
+Now imagine later in your program you want to say "Hello World!" five times
+again. We would have to output the value of `phrase` five times.
+
+If we wanted to use a method to contain this actions, it could look like this:
+
+```ruby
+def say_hello_world_five_times
+  phrase = "Hello World!"
+  puts phrase
+  puts phrase
+  puts phrase
+  puts phrase
+  puts phrase
+end
+```
+
+Now, when we use the word `say_hello_world_five_times` in our program, it
+will invoke the method, running the code within the method. So the script above,
+saying hello five times, doing other things, then saying hello five times again
+could be rewritten as this:
```

Comment:
the code below doesn't show the idea of "print 5, other work, print 5". Make the code align to this pitch.